slicedimage/_tileset.py

```diff
@@ -21,6 +22,20 @@ def __init__(
 
         self._discrete_dimensions = set()
 
+    def __repr__(self):
+        # get dimensions of optional shapes
+        shapes = (
+            "{k}: {v}".format(k=k, v=self.shape[k])
+            for k in self.dimensions - {'y', 'x'}
+        )
+        # get dimensions of x and y
```

Comment:
Keep in mind that default_tile_shape can be overridden by the individual tiles.  Would like an indication that it's just a fallback value.  Do we want to print out data on the individual tiles?  There might be an argument to do so given that coordinates lives on tiles.

Reply:
My gut reaction is that printing out information on individual tiles could get really big and may be hard to represent well for the variety of data types we have (e.g. 8 round 2 channel 1 z MERFISH vs 15 round, 2 channel, 90 z smFISH). 

I also suspect that at some point during analysis there's going to be a desire to reconcile coordinates so they match across the fov. 

I'll add a search for the overriding tile shape. Thanks for the catch.

Reply:
I'm guessing that some explicit method like `.dump()` might make sense to get detailed information.

Reply:
Do you want to take over this PR? :-) I think it'll be easier for you to push it across the finish line.